GitUI/CommandsDialogs/BrowseDialog/FormUpdates.Designer.cs

```diff
@@ -80,27 +69,40 @@ private void InitializeComponent()
             this.linkChangeLog.Visible = false;
             this.linkChangeLog.LinkClicked += new System.Windows.Forms.LinkLabelLinkClickedEventHandler(this.linkChangeLog_LinkClicked);
             // 
-            // btnDownloadNow
+            // btnUpdateNow
+            // 
+            this.btnUpdateNow.Enabled = false;
+            this.btnUpdateNow.Location = new System.Drawing.Point(410, 82);
+            this.btnUpdateNow.Name = "btnUpdateNow";
+            this.btnUpdateNow.Size = new System.Drawing.Size(140, 29);
+            this.btnUpdateNow.TabIndex = 4;
+            this.btnUpdateNow.Text = "Update Now";
+            this.btnUpdateNow.UseVisualStyleBackColor = true;
+            this.btnUpdateNow.Click += new System.EventHandler(this.btnUpdateNow_Click);
             // 
-            this.btnDownloadNow.Enabled = false;
-            this.btnDownloadNow.Location = new System.Drawing.Point(309, 82);
-            this.btnDownloadNow.Name = "btnDownloadNow";
-            this.btnDownloadNow.Size = new System.Drawing.Size(140, 29);
-            this.btnDownloadNow.TabIndex = 4;
-            this.btnDownloadNow.Text = "Download Now";
-            this.btnDownloadNow.UseVisualStyleBackColor = true;
-            this.btnDownloadNow.Click += new System.EventHandler(this.btnDownloadNow_Click);
+            // linkDirectDownload
+            // 
+            this.linkDirectDownload.AutoSize = true;
+            this.linkDirectDownload.Location = new System.Drawing.Point(220, 89);
+            this.linkDirectDownload.Margin = new System.Windows.Forms.Padding(4, 0, 4, 0);
+            this.linkDirectDownload.Name = "linkDirectDownload";
+            this.linkDirectDownload.Size = new System.Drawing.Size(223, 32);
+            this.linkDirectDownload.TabIndex = 3;
+            this.linkDirectDownload.TabStop = true;
+            this.linkDirectDownload.Text = "Direct Download";
+            this.linkDirectDownload.Visible = false;
+            this.linkDirectDownload.LinkClicked += new System.Windows.Forms.LinkLabelLinkClickedEventHandler(this.linkDirectDownload_LinkClicked);
             // 
             // FormUpdates
             // 
             this.AutoScaleDimensions = new System.Drawing.SizeF(120F, 120F);
```

Comment:
It looks like the last changes were applied in a scaled Windows.
We need to fix it and run all designers at 96dpi.

If you're not running Windows at 100% scale, please start your VS with `/noscale` switch.

Reply:
Ahh yes, I tried to keep everything the same. Looks like this slipped through as i made the changed on my laptop which has a high DPI screen. I'll correct this!

Reply:
Looks like you corrected this with your UI tweak 👍